src/lava/magma/runtime/message_infrastructure/test/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  GIT_REPOSITORY https://github.com/google/googletest.git
+  GIT_TAG release-1.12.1
+)
+
+# For Windows: Prevent overriding the parent project's compiler/linker settings
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(googletest)
+
```

Comment:
create a sub-directory under __test__ for cpp unit tests as following:
```
message_infrastructure
| - test \
      | - cpptest\*.cc
```

Reply:
Resolved in commit [6cea024](https://github.com/hongdami/lava/pull/5/commits/6cea02411f6255e0ed30146afca7bbe9163224f3)

Reply:
@seowwj pls push the commit under this PR.